src/hooks/useTokenCirculation.ts

```diff
@@ -10,7 +10,18 @@ export interface StatsData {
 }
 
 export function useTokenCirculation() {
-  const { currentNetworkName } = useNetworkInfo();
+  const { currentNetworkName, isZkEvm, dappStakingCurrency } = useNetworkInfo();
+  const currencyToNetworkMap: Record<string, string> = {
```

Comment:
I think you can use `networkNameSubstrate` variable from `useNetworkInfo`, so that you don't need to create `currencyToNetworkMap` variable.

![image](https://github.com/AstarNetwork/astar-apps/assets/92044428/6be1246d-de94-4ff0-bc58-ef75f15afe33)

Reply:
ohhhhh I didn't recognized it, I will refactor it while update zKatana to Kyoto

Reply:
resolved with [932feab](https://github.com/AstarNetwork/astar-apps/pull/1239/commits/932feab223da62216a7166f34805d3b3dd978b17)